woocommerce/woocommerce
# CYS: Remove not necessary patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR removes all the unnecessary patterns. We decided to keep in the plugin only the pattern necessary for the default CYS experience. 

For the sake of simplicity, for the AI flow, `template1` will always be used for the homepage (same beavhiour of the no-ai flow) 

Part of #47402.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:



1. Head over to WooCommerce > Home > Customize your store.
2. Click on `Start Designing`.
3. Click on Design Your Homepage.
4. Ensure that three patterns are visible.
5. Add them and ensure that images are loaded correctly.
6. Go back.
7. Click on "Choose your header".
8. Add them and ensure that images are loaded correctly.
9. Ensure that four patterns are visible.
10. Go back.
11. Click on "Choose your footer".
12. Add them and ensure that images are loaded correctly.
13. Ensure that three patterns are visible.




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Remove not necessary patterns.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
